src/clr-core/tsconfig.spec.json

```diff
@@ -1,11 +1,9 @@
 {
-  "extends": "./tsconfig.json",
+  "extends": "./tsconfig.project.json",
   "compilerOptions": {
-    "outDir": "../../out-tsc/spec",
-    "types": ["jasmine", "node"],
-    "target": "es2015",
-    "noImplicitAny": false
+    "outDir": "../../dist/clr-core",
+    "types": ["jasmine"],
+    "strictNullChecks": false
```

Comment:
I left this as false for the unit tests as it was a lot of work to change. Theres checks due to dom queries always possibly being null in the return signature.